AllReadyApp/Web-App/AllReady/Areas/Admin/ViewModels/Validators/Task/TaskEditViewModelValidator.cs

```diff
@@ -53,22 +53,22 @@ public TaskEditViewModelValidator(IMediator mediator)
             }
             
             // Rule - Attachments are optional
-            if (viewModel.NewAttachment != null && !string.IsNullOrEmpty(viewModel.NewAttachment.Name))
+            if (viewModel.NewAttachment != null && !string.IsNullOrEmpty(viewModel.NewAttachment.FileName))
             {
                 // Rule - New attachment must have content
-                if (viewModel.NewAttachment.Content == null || viewModel.NewAttachment.Content.Length == 0)
+                if (viewModel.NewAttachment == null || viewModel.NewAttachment.Length == 0)
                 {
                     result.Add(new KeyValuePair<string, string>(nameof(viewModel.NewAttachment), "The attachment is empty"));
                 }
 
                 // Rule - New attachment must have a maximum size
-                if (viewModel.NewAttachment.Content.Length > MaxAttachmentBytes)
+                if (viewModel.NewAttachment.Length > MaxAttachmentBytes)
```

Comment:
I left a note in a later commit about MaxAttachmentBytes. Please see that note. If this is not a blob storage size forced on us by Azure, I think we can remove it.